samples/highcharts/accessibility/accessible-dynamic-data/demo.js

```diff
@@ -0,0 +1,61 @@
+const chart = Highcharts.chart('container', {
+    title: {
+        text: 'Dynamic data'
+    },
+    subtitle: {
+        text: 'Click button to animate or explore chart'
+    },
+    accessibility: {
+        enabled: true
```

Comment:
Since this is default I'd leave it out of the demo - also for testing purposes to make sure this works with default config